.github/workflows/validate.yml

```diff
@@ -1,68 +1,71 @@
----
-name: Go
-on:
-  pull_request:
-  push:
-    branches:
-      - main
-      - "release-*"
-
-# Modified to avoid canceling all matrix jobs when one fails
-# Each job type will have its own concurrency group
-concurrency:
-  group: ${{ github.workflow }}-${{ github.job }}-${{ (github.event.pull_request && github.event.pull_request.number) || github.ref || github.run_id }}
-  cancel-in-progress: true
-
-# Minimal permissions to be inherited by any job that don't declare it's own permissions
-permissions:
-  contents: read
-
-jobs:
-  supportedVersions:
-    name: Fetch supported Go versions
-    runs-on: ubuntu-latest
-    outputs:
-        supported_versions: ${{ steps.matrix.outputs.supported_versions }}
-    steps:
-      - name: Checkout code
+---
+name: Validate
```

Comment:
Can we keep Go name (also in file)? Otherwise we need to change GH CI required checks and generally change the experience of contributing to client_golang. Is it worth the name change?